docs/src/content/docs/quickstart/sql-overview.mdx

```diff
@@ -0,0 +1,636 @@
+ name        | :utf8                                                                                                                                       | xtdb          | people     | public
+ likes       | [:union #{[:list [:union #{:f64 :utf8 [:struct {dynamic :utf8}]}]] :null}]                                                                  | xtdb          | people     | public
+ info        | [:union #{[:struct {contact [:union #{:null [:list [:struct {loc :utf8, registered [:union #{[:date :day] :null}], tel :utf8}]]}]}] :null}] | xtdb          | people     | public
+(8 rows)
+```
+  </Card>
+  </TabItem>
+</Tabs>
+
+Captureing fast-changing data may be powerful, but what if we made a mistake somewhere and wanted to undo a change? The thing that makes XTDB _most_ interesting is the approach to immutability and time-travel...
+
+## Query the past
+
+XTDB automatically records all versions and changes to individual rows across your database. This is widely useful for building reliable information systems, and it all starts with a [log](https://engineering.linkedin.com/distributed-systems/log-what-every-software-engineer-should-know-about-real-time-datas-unifying).
+
+### A log of transactions
+
+Alongside an HTTP API, XTDB supports a [Postgres wire protocol](/drivers/postgres/getting-started) compatibility endpoint that enables developers to re-use existing tools and drivers that were built for connecting to real Postgres servers.
+
+A key distinction between interacting with Postgres and interacting with XTDB (e.g. using `psql` or otherwise) is that all clients connected to XTDB operate in a 'stateless' manner that forces all writes to be fully serialized into a single, system-wide log of durably-recorded transactions.
+
+You can easily explore the detailed history of prior transactions with a "recent transactions" query against the system-maintained `_txs` table:
+
+<Tabs syncKey="psql">
+  <TabItem label="XT Play" icon="seti:db">
+    <Fiddle magicContext="my-context">
+      <Query q={`
+SELECT * FROM _txs ORDER BY _id DESC LIMIT 20`} />
+    </Fiddle>
+  </TabItem>
+  <TabItem label="psql" icon="laptop">
+  <Card>
+```sql
+user=> SELECT * FROM _txs ORDER BY _id DESC LIMIT 20;
+ _id  | committed | error |            tx_time
+------+-----------+-------+-------------------------------
+ 2722 | t         |       | 2024-07-15T12:43:27.345281Z
+ 1341 | t         |       | 2024-07-15T12:38:12.750543Z
+    0 | t         |       | 2024-07-15T12:36:31.310430Z
+(3 rows)
+```
+  </Card>
+  </TabItem>
+</Tabs>
+
+A key benefit of this log-oriented design is that queries can use the reliable ordering of the log to query the previous state of the database using nothing more than a past timestamp, as we will see next. This makes querying across a scaled-out cluster of consistent read very simple. For more information on the implications of the log-oriented design, see [How XTDB works](/intro/data-model).
```

Comment:
```
This approach means that queries can use the **reliable ordering** of the log
to query **previous states** of the database using nothing more than a timestamp.
More on that next!

This also makes querying across a **scaled-out cluster** very simple. 
For more information on the implications of the log-oriented design, see [How XTDB works](/intro/data-model).
```

Reply:
This bit is written as a benefit to the XTDB core developer, not to an end user.
Does it make it faster? More reliable?